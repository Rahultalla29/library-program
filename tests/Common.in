COMMON 100000 111111
ADD MEMBER Naruto Uzumaki
COMMON 100000 111111
ADD COLLECTION sample1.csv
ADD MEMBER Baruto Uzumaki
ADD MEMBER Minato Uzumaki
RENT 100000 111111
RENT 100000 111112
RENT 100000 111114
RENT 100000 111116
RENT 100000 111119
RENT 100000 111120
RELINQUISH ALL 100000
RENT 100001 111111
RENT 100001 111113
RENT 100001 111117
RENT 100001 111119
RENT 100001 111118
RENT 100001 111120
RELINQUISH ALL 100001
RENT 100002 111112
RENT 100002 111114
RENT 100002 111118
RENT 100002 111119
RENT 100002 111120
RENT 100002 111119
RENT 100002 111123
RENT 100002 111124
RENT 100002 111125
RELINQUISH ALL 100002
COMMON 100000 100001 100002 100001
COMMON 100000 100001 100002
EXIT


