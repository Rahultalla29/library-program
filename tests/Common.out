user: No members in system.

user: Success.

user: No books in system.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Book is currently unavailable.

user: Success.

user: Success.

user: Success.

user: Success.

user: Duplicate members provided.

user: The Odyssey (Homer)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)

user: Ending Library process.
